onnx2kerastl/utils.py

```diff
@@ -3,6 +3,24 @@
 from keras_data_format_converter import convert_channels_first_to_last
 import tensorflow as tf
 
+ONNX_ELEM_TO_TF_TYPE = {
```

Comment:
where do you use it?

Reply:
Don't currently use it but I keep it here since it's not well documented and we may want to enforce it sometime.